c/radmin.c

```diff
@@ -180,57 +259,96 @@ static void invokeRadmin(
   int oldKey = setKey(0);
 #endif
 
-  IRRSEQ00(
-      workArea,
-      safReturnCodeALET, safReturnCode,
-      racfReturnCodeALET, racfReturnCode,
-      racfReasonCodeALET, racfReasonCode,
-      functionCode,
-      parmList,
-      userID,
-      acee,
-      resultSubpool,
-      resultPtr
-  );
+  __asm(ASM_PREFIX
```

Comment:
Would it be possible to link radmin API the old way to get the entry point and then invoke that entry point via an inline HLASM? That was there won't be any questions in case the below isn't common knowledge.